plugins/GigPlayer/GigPlayer.cpp

```diff
@@ -1182,21 +1151,24 @@ void GigSample::updateSampleRate()
 
 
 
-bool GigSample::convertSampleRate( SampleFrame & oldBuf, SampleFrame & newBuf,
+bool GigSample::convertSampleRate(std::vector<SampleFrame>& oldBuf, std::vector<SampleFrame>& newBuf,
 		f_cnt_t oldSize, f_cnt_t newSize, float freq_factor, f_cnt_t& used )
 {
 	if( srcState == nullptr )
 	{
 		return false;
 	}
 
-	SRC_DATA src_data;
-	src_data.data_in = &oldBuf[0];
-	src_data.data_out = &newBuf[0];
-	src_data.input_frames = oldSize;
-	src_data.output_frames = newSize;
-	src_data.src_ratio = freq_factor;
-	src_data.end_of_input = 0;
+	SRC_DATA src_data{
+		/* .data_in */ reinterpret_cast<const float*>(oldBuf.data()),
+		/* .data_out */ reinterpret_cast<float*>(newBuf.data()),
```

Comment:
```suggestion
		/* .data_in */ oldBuf[0].data(),
		/* .data_out */ newBuf[0].data(),
```
I believe this is possible now

Reply:
I don't understand this too